src/verifyAuthorizationRequestCallback.ts

```diff
@@ -18,7 +20,7 @@ export const verifyAuthorizationRequestCallback = async ({
       }
 
       const jwsService = agentContext.resolve(JwsService)
-      const { isValid } = await jwsService.verifyJws(agentContext, { jws: va.data })
+      const { isValid } = await jwsService.verifyJws(agentContext, { jws: va.data, trustedCertificates })
```

Comment:
This doesn't have the changes yet that we allow any certificate and return whether it's trusted rigth?